### PR TITLE
Add radio buttons

### PR DESCRIPTION
**Pull request by EllenWiderstrand**

### CODE REVIEW

File: src/components/controls/radios/DefaultRadio/DefaultRadio.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    background: ${({ theme }) => theme.colors.background.secondary};
+  }
+`;
+
+export interface DefaultRadioProps
+  extends InputHTMLAttributes<HTMLInputElement> {
+  onChange: (e: ChangeEvent<HTMLInputElement>) => void;
+}
+
+export const DefaultRadio = ({ ...props }: DefaultRadioProps) => {
```
Comment (filiptammergard) [inline src/components/controls/radios/DefaultRadio/DefaultRadio.tsx:36]: I wonder if this DefaultRadio is needed. Would it be possible to just have a Radio, which result in a DefaultRadio if label is not passed, and LabelRadio if label is passed. What do you think?

Comment (filiptammergard) [inline src/components/controls/radios/DefaultRadio/DefaultRadio.tsx:36]: Instead of destructuring and spreading you can do just `props` 😊

Comment (EllenWiderstrand) [reply src/components/controls/radios/DefaultRadio/DefaultRadio.tsx:36]: I can try that out! 😊

**Review by filiptammergard — APPROVED** [review]
👍

**Pull request merged**